tests/applications.js

```diff
@@ -0,0 +1,54 @@
+import test from 'ava';
+import _ from 'lodash';
+
+import applications from '../lib/applications';
+import merged from './fixtures/applications/objects/model-merged.js';
+
+const reqObj = {
```

Comment:
Two things:
* Can this be renamed `req` as it doesn't conflict with anything else
* This is only used in one place, can it be moved to the one function it's used in?